src/verify_cert.rs

```diff
@@ -426,6 +429,61 @@ fn check_basic_constraints(
     }
 }
 
+/// Additional context for the `RequiredEkuNotFoundContext` error variant.
+///
+/// The contents of this type depend on whether the `alloc` feature is enabled.
+#[derive(Clone, PartialEq, Eq)]
+pub struct RequiredEkuNotFoundContext {
+    /// The required ExtendedKeyUsage.
+    #[cfg(feature = "alloc")]
+    pub required: KeyUsage,
+    /// The ExtendedKeyUsage OIDs present in the certificate.
+    #[cfg(feature = "alloc")]
+    pub present: Vec<Vec<usize>>,
+}
+
+impl fmt::Debug for RequiredEkuNotFoundContext {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let mut builder = f.debug_struct("RequiredEkuNotFoundContext");
+        #[cfg(feature = "alloc")]
+        builder.field(
+            "required",
+            match &self.required.inner {
+                ExtendedKeyUsage::Required(inner) => inner,
```

Comment:
> src/verify_cert.rs#L452
> Added line #L452 was not covered by tests

I think the `Required` branch here would be an easy coverage fix if you're interested.

Reply:
Hmmm, my quick fix doesn't seem to have helped. Going to skip this for now...